platformio.ini

```diff
@@ -1,22 +1,26 @@
 ; This file allows the docker build file to install the required platformio
 ; platforms
 
-[env:espressif8266-2.4.2]
-platform = espressif8266@1.8.0
```

Comment:
Framework v2.4.2 for ESP8266 has been removed (was mainly used for the lint image).

Instead there's now a special lint image.